src/js/common/browserInfoHelper.js

```diff
@@ -68,7 +67,19 @@ const validateAndGetScreenDetail = (value) => {
   }
 };
 
-export const getTimezone = () => `UTC${moment().format("Z")}`;
+const getFormattedOffset = () => {
+    const nowUTC = new Date().toString();
+    // nowUTC is in format like "Wed Sep 30 2020 23:11:02 GMT+0100 (British Summer Time)"
+    // To format the offset, we pick the relevant characters based on their position
+    // and reformat with a ":"
+    const signCharIndex = 28;
```

Comment:
I was thinking figuring out position 28, 29 etc; from such long string is difficult. How about splitting the string to bare minimum required (i.e +0100)  and applying the logic to fetch hour and minute offset from that?

Eg: const nowUTC = new Date().toString().split('GMT')[1].split(' ')[0]; // gives "+0100"

Reply:
if we're just reading the timestamp from the current browser session, it's possible to use [getTimezoneOffset](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset)?

```javascript
const offset = new Date().getTimezoneOffset() * -1;
const offsetHours = `${Math.floor(offset / 60)}`.padStart(2, 0);
const offsetMinutes = `${Math.abs(offset % 60)}`.padStart(2, 0);
const sign = offset >= 0 ? '+' : '-'
const offsetString = `UTC${sign}${offsetHours}:${offsetMinutes}`

```

Reply:
> if we're just reading the timestamp from the current browser session, it's possible to use [getTimezoneOffset](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset)?
> 
> ```js
> const offset = new Date().getTimezoneOffset() * -1;
> const offsetHours = `${Math.floor(offset / 60)}`.padStart(2, 0);
> const offsetMinutes = `${Math.abs(offset % 60)}`.padStart(2, 0);
> const sign = offset >= 0 ? '+' : '-'
> const offsetString = `UTC${sign}${offsetHours}:${offsetMinutes}`
> ```

While this would work it looks a good deal more complicated than the current version in the PR, which is based on @skodamarthi 's suggestion.